htdocs/bom/class/bom.class.php

```diff
@@ -1806,6 +1785,7 @@ public function __construct(DoliDB $db)
 		global $conf, $langs;
 
 		$this->db = $db;
+		$this->setPicto('bomline');
```

Comment:
What is the goal to have this compared to old code ?

Reply:
regarding to PHP 8.2 documentation it seems to be the best way to not overrid the property $picto of the common object line .
if in the __construct method you just call $this->picto = 'bomline' PHP consider this as on override.